g3doc/quick_reference.md

```diff
@@ -953,6 +957,80 @@ encoding depends on the platform).
     next byte. Returns the number of bytes written: `(Lanes(D()) + 7) / 8`. At
     least 8 bytes of `p` must be writable.
 
+*   <code>Mask&ltDTo&gt; **PromoteMaskTo**(DTo d_to, DFrom d_from,
+    Mask&ltDFrom&gt; m)</code>: Promotes `m` to a mask with a lane type of
```

Comment:
Typo: missing ; after several occurrences of `&lt`.

Reply:
I have already replaced `&lt` with `&lt;` in the documentation.